.github/workflows/docker.yml

```diff
@@ -0,0 +1,16 @@
+name: Build CLI Docker image
+
+on:
+  push:
+    paths:
+      - 'Dockerfile'
+
+jobs:
+  verify_image:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Build Image
+        run: docker buildx build --no-cache -t atlas .
```

Comment:
Can we run docker linter as part of the action? https://github.com/RedCoolBeans/dockerlint

Reply:
I'll look into that

Reply:
Thanks for the suggestion, I have added hadolint to the pipeline